src/blocks/advanced-form/fields/radio/edit.js

```diff
@@ -223,6 +223,11 @@ function FieldRadio( { attributes, setAttributes, isSelected, clientId, context,
 								>
 									{__( 'Add Option', 'kadence-blocks' )}
 								</Button>
+								<ToggleControl
+									label={__( 'Show inline?', 'kadence-blocks' )}
```

Comment:
I like "display inline?" better for this label.
I think "show", for me, implies revealing something that wasn't there before.

Reply:
Done